_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,353 @@
+  with modular, extensible, and peer-reviewed components.  A longer
+  term ambition is to provide a platform for building native
+  wallets and integrating them with existing projects.  Filini demoed
+  the current functionality, which includes a playground with a Policy
+  to [Miniscript][topic miniscript] compiler and some rudimentary
+  visualizations. It is written in Rust and leverages the
+  rust-miniscript library and the open source Esplora block explorer.
+  ([transcript][mb ts], [video][mb vid])
+
+- **Watchtowers and BOLT13:** Sergi Delgado appeared on [Potzblitz][] to
+  discuss the latest state of [watchtower][topic watchtowers]
+  development and a proposed watchtower protocol specification. He
+  explored the various tradeoffs when designing a watchtower and the
+  interplay between privacy requirements, accessibility, storage, and
+  fees charged.  Delgado is working on the watchtower implementation
+  [The Eye of Satoshi][teos] at Talaia Labs, which is aiming to be
+  compliant with the proposed BOLT13 (see [Newsletter #75][news75
+  watchtower bolt]).  Delgado also highlighted how watchtowers are
+  becoming increasingly critical in multiple settings such as Bitcoin
+  [vault][topic vaults] designs, statechains, and atomic swaps in addition to LN.
+  ([transcript][watchtowers ts], [video][watchtowers vid],
+  [slides][watchtowers slides])
+
+- **CoinSwap:** Aviv Milner presented on CoinSwap at the Wasabi Research
+  Club. He explained the property of covertness and how Chris Belcher’s
+  CoinSwap proposal (see [Newsletter #100][news100 coinswap]) provides
+  covertness in a manner that other privacy schemes such as
+  [coinjoin][topic coinjoin] fail to do. Milner also went through the
+  motivation for routing CoinSwaps, namely to address the concern of
+  unwittingly entering into a CoinSwap with an adversary such a chain
+  surveillance company.  ([transcript][coinswap ts], [video][coinswap
+  vid])
+
+- **Schnorr signatures and multisignatures:** [BIP340][] co-authors Tim
+  Ruffing, Pieter Wuille, and Jonas Nick held a discussion at London
+  Bitcoin Devs on the specification of [schnorr signatures][topic
+  schnorr signatures].  This covered earlier ideas for implementing
+  schnorr signatures in Bitcoin and what the co-authors thought the
+  community should be concerned about when considering a possible future
+  soft fork deployment. Wuille noted that he is most concerned about how
+  schnorr is adopted and what is built using it.  The following day, Tim
+  Ruffing presented on the challenges of multisignature and threshold
+  signature schemes using schnorr signatures.  He's been working with
+  collaborators on a speculative [MuSig][topic musig] signature scheme
+  that only requires two rounds of interaction rather than three,
+  without relying on zero knowledge proofs, which would greatly simplify
+  some threshold signing workflows.  ([Meeting transcript][socratic ts],
```

Comment:
I would prefer to revert the change made to this Schnorr section as I think the original was stronger. First the speculative MuSig scheme doesn't just impact threshold signing workflows it also impact multisignature workflows. I also thought the original discussion of the added complexities on threshold over multisig was interesting.

Reply:
It's your call.  Would you mind if I give it another try to keep it short but add a link to https://bitcoinops.org//en/newsletters/2019/10/16/#the-quest-for-practical-threshold-schnorr-signatures where we previously described his concerns?

Reply:
Sure that link is a good one. In that case "simplify some **multisignature and** threshold signing workflows"?